README.md

```diff
@@ -1,70 +1,62 @@
-# Getting Started with Create React App
+# _Pokemon Card Database_
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+A frontend app that links to an API of Pokemon Cards that you can view or add to personal collections.
 
-## Available Scripts
+---
 
-In the project directory, you can run:
+## Authors
 
-### `npm start`
+Kortney Foley
 
-Runs the app in the development mode.\
-Open [http://localhost:3000](http://localhost:3000) to view it in your browser.
+---
 
-The page will reload when you make changes.\
-You may also see any lint errors in the console.
+## Technologies Used
 
-### `npm test`
+-   HTML
+-   JavaScript
+-   CSS
+-   React
+-   [Axios](https://www.npmjs.com/package/axios)
+-   [React Query](https://tanstack.com/query/v4/?from=reactQueryV3&original=https://react-query-v3.tanstack.com/)
+-   [React Router](https://reactrouter.com/en/main)
 
-Launches the test runner in the interactive watch mode.\
-See the section about [running tests](https://facebook.github.io/create-react-app/docs/running-tests) for more information.
+---
 
-### `npm run build`
+## Description
 
-Builds the app for production to the `build` folder.\
-It correctly bundles React in production mode and optimizes the build for the best performance.
+The purpose of designing this app was to develop my back end and database creation skills. Original version of the app had full CRUD functionality so card could be updated, added or deleted, but it didn't make sense to keep all these functions for general users, so most of these have been removed or limited to "admin" account user.
```

Comment:
```suggestion
The purpose of designing this app was to develop my back end and database creation skills. Original version of the app had full CRUD functionality so that a card could be updated, added or deleted, but it didn't make sense to keep all these functions for general users, so most of these have been removed or limited to "admin" account user.
```